syntax = "proto3";
package Protobuf;
option csharp_namespace = "Communication.Proto";

import "MessageType.proto";
import "Message2Client.proto";

message MessageToServer
{
    MessageType messageType = 1;
    int64 playerID = 2;		//消息发送者的playerID
    int64 teamID = 3;		//消息发送者所在队伍的ID
    JobType jobType = 4;	//messageType为AddPlayer时选择的职业
    PropType propType = 5;	//messageType为Pick时要捡起的道具类型
    int32 timeInMilliseconds = 6;	//时间参数
    double angle = 7;		//角度参数
    int64 ToPlayerID = 8;	//当messageType为Send时有效，为发送对象的ID
    string message = 9;		//当messageType为Send时有效，为发送的消息
}
